lib/solidus_paypal_braintree/engine.rb

```diff
@@ -20,9 +20,12 @@ class Engine < Rails::Engine
 
     initializer "register_solidus_paypal_braintree_gateway", after: "spree.register.payment_methods" do |app|
       app.config.spree.payment_methods << SolidusPaypalBraintree::Gateway
+      SolidusPaypalBraintree::Gateway.allowed_admin_form_preference_types << :preference_select
       Spree::PermittedAttributes.source_attributes.concat [:nonce, :payment_type, :paypal_funding_source]
     end
 
+    config.assets.precompile += ['solidus_paypal_braintree_manifest.js']
```

Comment:
is this for backend or frontend, maybe we can add below in the already existing `config.assets.precompile` lists?

Reply:
The images are only used for frontend currently, so I will move it appropriately, thanks! 👍